### DISCUSSION TITLE
Add articles to "consumed" and subfolders, and a "read" flag

### DISCUSSION THREAD

**Discussion by michael-vigato** [dailydotdev/daily#503] [Feature Requests]
I like to keep a stash of articles I've read, so I can go back to them in the future, forward them to other people, or just re-read them in the future

It would be very nice to be able to flag each article with a "read" flag (as in past tense of "read"), and possibly be able to store them in an "archive" folder for easy future access

(I'm aware there is a "reading history" feature, but I'm referring to a personally curated archive, that can be organized and sorted, and where I only put articles I'm interested in coming back in the future)

**Comment by idoshamun** [comment]
Hi @vgtmhl,

We used to have a paid feature of bookmark lists but we deprecated it when we shut down our premium plan.
We are considering relaunching this feature as a free for all. But no concrete plans yet.
Thanks for the suggestion :)
